test/specs/debugApexTests.e2e.ts

```diff
@@ -111,36 +110,56 @@ describe('Debug Apex Tests', async () => {
   });
 
   step('Debug all Apex Methods on a Class via the Test Sidebar', async () => {
-    const workbench = await browser.getWorkbench();
-    const testingView = await workbench.getActivityBar().getViewControl('Testing');
+    const workbench = await (await browser.getWorkbench()).wait();
+    await utilities.runCommandFromCommandPrompt(workbench, 'Testing: Focus on Apex Tests View', 1);
 
     // Open the Test Sidebar
-    const testingSideBarView = await testingView?.openView();
-    expect(testingSideBarView).toBeInstanceOf(SideBarView);
-
     const sidebar = workbench.getSideBar();
     const sidebarView = sidebar.getContent();
     const apexTestsSection = await sidebarView.getSection('APEX TESTS');
     expect(apexTestsSection.elem).toBePresent();
 
+    let apexTestsItems = (await apexTestsSection.getVisibleItems()) as TreeItem[];
+    await browser.keys(['Escape']);
+
+    // If the Apex tests did not show up, click the refresh button on the top right corner of the Test sidebar
```

Comment:
As we're using this also in runApexTests.e2e.ts, what do you think about moving this logic to a new method in apexUtils.ts?